src/DynamoCore/Graph/Nodes/PortModel.cs

```diff
@@ -440,6 +440,40 @@ internal string GetInputPortType()
             }
             return null;
         }
+
+        /// <summary>
+        /// Returns the string representation of the fully qualified typename
+        /// where possible for the port if it's an output port. This method currently
+        /// returns a valid type for only Zero Touch, Builtin and NodeModel nodes,
+        /// and returns null otherwise. The string representation of the type also
+        /// contains the rank information of the type, e.g. Point[], or var[]..[]. 
+        /// </summary>
+        /// <returns>output port type</returns>
+        internal string GetOutPortType()
+        {
+            if (PortType == PortType.Input) return null;
+
+            var ztNode = Owner as DSFunction;
```

Comment:
instead of doing type checking like this using as and var you can probably switch to:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/is#pattern-matching-with-is

maybe a little cleaner.

Reply:
Ah I can see you are following the same approach as `GetInPortType`, do you mind correct both in your PR?
```
 If (ztNode is DSFunction)
```

Reply:
Changed this to use the "is" operator